students/vkis/lesson04/kata14.py

```diff
@@ -0,0 +1,200 @@
+# Lesson 04 - Kata14
+# Objective: look at all 3 adjacent words in a textfile,
+# treat the first 2 as KEYS and the last as a VALUE
+# try to construct the longest string-of-combinations 
+# =================================================================
+
+# plug-ins
+import os
+
+# declarations
+dict = {}
+
+
+
+
+def formatter(myline):
+    # clean-up text for splitting
+    myline = myline.replace("-", " ")
+    myline = myline.replace(",", " ")
+    myline = myline.replace("(", " ")
+    myline = myline.replace(")", " ")
+    myline = myline.replace("\n", " ")
+    return myline
```

Comment:
i think this could be re-written as a one liner but not sure if you would want to hehe.